pantheon-bundle/frontend/src/app/search.tsx

```diff
@@ -302,6 +309,11 @@ class Search extends Component<IAppState, ISearchState> {
       </React.Fragment>
     );
 
+    const editMetadataAlert = (
```

Comment:
Is this ever used? If not we should delete

Reply:
removed!